packages/chip/src/__tests__/Chip.screenshots.test.tsx

```diff
@@ -0,0 +1,32 @@
+import {
+  createStorybookUrl,
+  setupScreenshotTesting,
+} from '../../../screenshotUtils';
+
+const packageName = 'chip';
+const chipLabel = 'Chip CTA';
+const chipVariants = ['filter', 'choice'];
+const clip = { x: 0, y: 0, width: 200, height: 100 };
+const screenshotTesting = setupScreenshotTesting({
+  it,
+  beforeAll,
+  afterAll,
+  expect,
+});
+
+describe(
+  'Chip variants',
+  screenshotTesting({
+    cases: chipVariants.map(variant => [
+      'sprite',
+      createStorybookUrl({
+        packageName,
+        knobs: {
+          variant,
+          children: chipLabel,
+        },
+      }),
+    ]),
+    screenshotOpts: { clip },
+  }),
+);
```

Comment:
Where is checks for all of behaves of the component  Default Hover Pressed Disable?

Reply:
added